### ISSUE TITLE
Rounded Rectangle Feature for Geometry Tool

### ISSUE THREAD

**Issue by erayzesen** [tahoma2d/tahoma2d#1290]
**Is your feature request related to a problem? Please describe**
Geometry Tool is used at all levels. I particularly seek the rounding of corners of rectangles at vector levels; it's a must-have for vector software. With this feature, I can easily reapply my vector designs in Tahoma2D. 

**Describe the solution you'd like**
When selecting a rectangle from the Geometry tool, an additional setting may appear on the Tool Option Bar for its implementation.

**Comment by manongjohn** [comment]
For Vector strokes the Geometry and Selection tools have the following options available for changing end caps and miter joins:

![image](https://github.com/tahoma2d/tahoma2d/assets/19245851/7730d794-7bab-4682-820f-b19c149a5963)

Are you looking for something more than this?  If so, please explain. If there is an example from other software, please provide video/image reference.

**Comment by erayzesen** [comment]
Of course, It needs a little bit explain. I also know these options. But these're about with the stroke connection.

It makes to me: 
![image](https://github.com/tahoma2d/tahoma2d/assets/65820625/50472ad1-fcf5-4daa-b7c7-49432073a9dd)

I need this : (inkscape)
https://github.com/tahoma2d/tahoma2d/assets/65820625/557e2880-61b5-45c7-b456-785c4efa77de

Inkscape gives us x and y options but actually only radius option is enough to general use. For example Adobe animate uses "rectangle corner radius" feature for this. 
https://youtu.be/Sspl01q2n5M?si=upUq36TEeCPh55l1&t=353

The caps and mitter options can be hack to creating a solid rounded rectangle with using the same color for stroke and fill color.  But we can't do it if we want to shapes with stroke.
